README.md

```diff
@@ -1,35 +1,44 @@
 # 🚖 **CabRental**
 
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![Last Commit](https://img.shields.io/github/last-commit/07sumit1002/CabRental.svg)](https://github.com/07sumit1002/CabRental/commits/main)
-[![GitHub Issues](https://img.shields.io/github/issues/07sumit1002/CabRental.svg)](https://github.com/07sumit1002/CabRental/issues)
-[![GitHub Pull Requests](https://img.shields.io/github/issues-pr/07sumit1002/CabRental.svg)](https://github.com/07sumit1002/CabRental/pulls)
-[![Platform](https://img.shields.io/badge/platform-Web%20%26%20Blockchain-green.svg)](#)
-[![Technologies](https://img.shields.io/badge/Tech-HTML%20%7C%20CSS%20%7C%20JS%20%7C%20Node.js%20%7C%20Solidity-blue.svg)](#)
+🚗 CabRental is a **decentralized vehicle rental platform** using **Web3** for enhanced privacy, security, and flexibility. With crypto-wallet authentication 🔐 and smart contract-based transactions 📜, it offers a seamless, transparent, and tamper-proof rental experience.
+
+
+<table align="center">
+    <thead align="center">
+        <tr border: 2px;>
+            <td><b>🌟 Stars</b></td>
+            <td><b>🍴 Forks</b></td>
+            <td><b>🐛 Issues</b></td>
+            <td><b>🔔 Open PRs</b></td>
+            <td><b>🔕 Close PRs</b></td>
+        </tr>
+     </thead>
+    <tbody>
+         <tr>
+            <td><img alt="Stars" src="https://img.shields.io/github/stars/07sumit1002/CabRental?style=flat&logo=github"/></td>
+             <td><img alt="Forks" src="https://img.shields.io/github/forks/07sumit1002/CabRental?style=flat&logo=github"/></td>
+            <td><img alt="Issues" src="https://img.shields.io/github/issues/07sumit1002/CabRental?style=flat&logo=github"/></td>
+            <td><img alt="Open Pull Requests" src="https://img.shields.io/github/issues-pr/07sumit1002/CabRental?style=flat&logo=github"/></td>
+           <td><img alt="Close Pull Requests" src="https://img.shields.io/github/issues-pr-closed/07sumit1002/CabRental?style=flat&color=critical&logo=github"/></td>
+        </tr>
+    </tbody>
+</table>
 
 ---
-![CabRental Banner](https://files.oaiusercontent.com/file-E72nIXce4E1497vZoiZtGNTy?se=2024-10-05T14%3A47%3A35Z&sp=r&sv=2024-08-04&sr=b&rscc=max-age%3D604800%2C%20immutable%2C%20private&rscd=attachment%3B%20filename%3Dff26116f-77dd-4e15-aee3-e867ef6fb7ac.webp&sig=L/j24P/8ksOZiXp9XSW7mCs1EKYRokZa%2BQjwzPtzb5s%3D)  
+<img src = "https://github.com/neeru24/Connect_icons/blob/main/Screenshot%202024-10-12%20105220.png" alt = "image">
+<img src = "https://github.com/neeru24/Connect_icons/blob/main/Screenshot%202024-10-12%20120929.png" alt = "image">
```

Comment:
Instead of using url from your github account, download images in the repo and use local url paths to show the images.

Reply:
> Instead of using url from your github account, download images in the repo and use local url paths to show the images.

Actually I was trying to do so, but it's showing some restrictions that's why, I did so.. Isn't it fine?